pykickstart/commands/snapshot.py

```diff
@@ -0,0 +1,96 @@
+#
+# Jiri Konecny <jkonecny@redhat.com>
+#
+# Copyright 2005, 2006, 2007, 2008, 2012, 2016 Red Hat, Inc.
```

Comment:
Ohh damn. Now I need to rewrite it completely with 2017 code style... 🤣 
Fixed.